{
  "vulinfoid": "JVNDB-2016-008385",
  "vulinfodata": {
    "title": "ImageMagick の coders/wpg.c におけるサービス運用妨害 (DoS) の脆弱性",
    "vulinfodescription": {
      "overview": "ImageMagick の coders/wpg.c には、サービス運用妨害 (境界外読み取り) 状態にされる脆弱性が存在します。"
    },
    "affected": {
      "affecteditem": [
        {
          "name": "ImageMagick",
          "productname": "ImageMagick",
          "cpe": {
            "text": "cpe:/a:imagemagick:imagemagick",
            "version": "2.2"
          }
        }
      ]
    },
    "impact": {
      "cvss": [
        {
          "version": "2.0",
          "severity": {
            "text": "Medium",
            "type": "Base"
          },
          "base": "4.3",
          "vector": "AV:N/AC:M/Au:N/C:N/I:N/A:P"
        },
        {
          "version": "3.0",
          "severity": {
            "text": "Medium",
            "type": "Base"
          },
          "base": "6.5",
          "vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H"
        }
      ],
      "impactitem": {
        "description": "リモートの攻撃者により、巧妙に細工されたファイルを介して、 サービス運用妨害 (境界外読み取り) 状態にされる可能性があります。"
      }
    },
    "solution": {
      "solutionitem": {
        "description": "ベンダより正式な対策が公開されています。ベンダ情報を参照して適切な対策を実施してください。"
      }
    },
    "related": {
      "relateditem": [
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "out-of-bounds read in MagickCore/memory.c:707:23 #122",
          "url": "https://github.com/ImageMagick/ImageMagick/issues/122"
        },
        {
          "type": "vendor",
          "name": "GitHub",
          "vulinfoid": "#122",
          "url": "https://github.com/ImageMagick/ImageMagick/commit/a251039393f423c7858e63cab6aa98d17b8b7a41"
        },
        {
          "type": "vendor",
          "name": "Launchpad",
          "vulinfoid": "Bug #1542115",
          "url": "https://bugs.launchpad.net/ubuntu/+source/imagemagick/+bug/1542115"
        },
        {
          "type": "vendor",
          "name": "Red Hat Bugzilla",
          "vulinfoid": "Bug 1378759",
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1378759"
        },
        {
          "type": "advisory",
          "name": "Common Vulnerabilities and Exposures (CVE)",
          "vulinfoid": "CVE-2016-7527",
          "url": "http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-7527"
        },
        {
          "type": "advisory",
          "name": "National Vulnerability Database (NVD)",
          "vulinfoid": "CVE-2016-7527",
          "url": "https://nvd.nist.gov/vuln/detail/CVE-2016-7527"
        },
        {
          "type": "cwe",
          "name": "JVNDB",
          "vulinfoid": "CWE-125",
          "url": "https://cwe.mitre.org/data/definitions/125.html",
          "title": "境界外読み取り"
        }
      ]
    },
    "history": {
      "historyitem": [
        {
          "historyno": "0",
          "datetime": "2018-02-17T10:37:52+09:00",
          "description": "[2017年05月17日]\\n  掲載"
        }
      ]
    },
    "datefirstpublished": "2017-05-17T12:00:11+09:00",
    "datelastupdated": "2017-05-17T12:00:11+09:00",
    "datepublic": "2016-02-05T00:00:00+09:00"
  }
}
